<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>292</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_commit">
      <property name="text">
       <string>commit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Raise">
      <property name="text">
       <string>Raise</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Down">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_Random">
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_Bet">
      <property name="text">
       <string>Bet</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Oppent_betting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_bet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_random">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_coin">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BET coin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total coin</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_gamecount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sad.qrc">:/new/prefix1/title.PNG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sad.qrc">:/new/prefix1/player.PNG</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PLayer_check</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_player">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>lcdNumber_bet</zorder>
  <zorder>lcdNumber_random</zorder>
  <zorder>lcdNumber_coin</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lcdNumber_gamecount</zorder>
  <zorder>label_5</zorder>
  <zorder>label_3</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>lcdNumber_player</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sad.qrc"/>
 </resources>
 <connections/>
</ui>
